R/pkg/R/DataFrame.R

```diff
@@ -851,6 +849,8 @@ setMethod("nrow",
             count(x)
           })
 
+#' ncol
```

Comment:
this doesn't seem consistent? in L713, or L675 we are changing doc title to something descriptive, yet here and below we are changing title to just the function name?